console_backend/src/status_bar.rs

```diff
@@ -274,6 +274,7 @@ impl StatusBar {
                 .heartbeat_data
                 .lock()
                 .expect(HEARTBEAT_LOCK_MUTEX_FAILURE);
+            shared_data.ins_status_string = format!("0x{:<01x}", msg.flags);
```

Comment:
basically how solution table inserts ins status string, did we want to replicate the logic from the solution table?